wolfcrypt/src/tfm.c

```diff
@@ -2956,15 +2956,18 @@ int mp_prime_is_prime_ex(mp_int* a, int t, int* result, WC_RNG* rng)
          * 2 times the prime size. */
         word32 baseSz;
         #ifndef WOLFSSL_SMALL_STACK
-            byte base[FP_MAX_BITS/32];
+            byte base[1+FP_MAX_BITS/16];
```

Comment:
Do we have a define/enum value that can be used here instead of `16`? Is the `+1` to round up? Can you add a comment here about the calculation of size?